pkg/helm/chart.go

```diff
@@ -141,6 +141,7 @@ func (c *ActionableChart) Install(
 	client := action.NewInstall(c.config)
 	client.ReleaseName = c.releaseName
 	client.Namespace = c.namespace
+	client.Wait = true
```

Comment:
I don't think we want to add this to the install at the moment since there are some scenarios where we really want this to be fire and forget.